packages/account/build/webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = function (env) {
         context: path.resolve(__dirname, '../src'),
         devtool: IS_RELEASE ? undefined : 'eval-cheap-module-source-map',
         entry: {
-            account: path.resolve(__dirname, '../src', 'index.js'),
+            account: path.resolve(__dirname, '../src', 'index.tsx'),
```

Comment:
Shouldn't be a `.ts` file?

Reply:
I've checked it is .tsx file cause it passes component inside